src/components/Navigator.js

```diff
@@ -28,11 +28,11 @@ class Navigator extends PureComponent {
     }
 
     return (
-      <div className={styles['root']}>
+      <div id='Navigator' className={styles['root']}>
         <div className={styles['content']}>
 
           <ul className={styles['tabs']}>
-            <li className={tabTwoClasses} onClick={onFriendClicked}>
+            <li className={tabTwoClasses} onClick={onFriendClicked} data-has-unread={friendListHasUnread}>
```

Comment:
we probably don't need this if we move everything to RootPage.js ?

Reply:
yeah, you're right